src/main/java/vazkii/botania/common/crafting/RecipeBrew.java

```diff
@@ -144,10 +144,7 @@ public boolean equals(Object o) {
 		public RecipeBrew read(@Nonnull ResourceLocation id, @Nonnull JsonObject json) {
 			String brewStr = JSONUtils.getString(json, "brew");
 			ResourceLocation brewId = ResourceLocation.tryCreate(brewStr);
-			if (brewId == null || !BotaniaAPI.instance().getBrewRegistry().getOptional(brewId).isPresent()) {
-				throw new JsonParseException("Unknown brew " + brewStr);
-			}
-			Brew brew = BotaniaAPI.instance().getBrewRegistry().getOrDefault(brewId);
+			Brew brew = BotaniaAPI.instance().getBrewRegistry().getOptional(brewId).orElseThrow(() -> new JsonParseException("Unknown brew " + brewStr));
```

Comment:
getOptional handles nulls?

Reply:
Signature in code is `public Optional<T> getOptional(@Nullable ResourceLocation id);`